src/test/scala/it/unibo/authsim/library/user/UserTest.scala

```diff
@@ -25,7 +25,7 @@ class UserTest extends AnyWordSpec{
   "A user" when {
     "created" should {
       "have a name" in{
-        assert(user.username == "Alexandra")
+        assert(user.username == "")
```

Comment:
Is this an error or is it wanted?

Reply:
I wanted to  test the email notification in case of failure

Reply:
I got it: it is the failing test for Travis emails